docs/assets/scripts/hedwig-main.umd.js

```diff
@@ -63,6 +63,44 @@
     var ascendingBisect = bisector(ascending);
     var bisectRight = ascendingBisect.right;
 
+    function extent(values, valueof) {
```

Comment:
Since you're already using d3 within hedwig I think this could be used https://github.com/d3/d3-array#extent

Reply:
Ahh... so this extent function is in fact the D3 version. What happens is  `rollup.config.js` is doing the bundling and a bit of tree shaking to bring in this function. It's not explicitly clear that its coming from `d3` but it definitely is.